source/documentation/reactivestreams-support.html.md.erb

```diff
@@ -0,0 +1,103 @@
+---
+title: Reactive Streams Support - ScalikeJDBC
+---
+
+## Reactive Streams Support
+
+<hr/>
+For ScalikeJDBC 3.0 or higher, we support the Publisher of Reactive Streams to subscribe results from database queries.
```

Comment:
>subscribe results from database queries

nit: each stream in the library comes from a single query. So, I think `subscribe a stream from a database query` is better.